lib/core/network/api/services/remote_config_service.dart

```diff
@@ -104,23 +104,23 @@ class RemoteConfigService {
   String get accountCreatorEndpoint => FirebaseRemoteConfig.instance.getString('accountCreatorEndpoint');
 
   // PPP Service AWS
```

Comment:
I guess this should be a single network specific object with all the ppp parameters in it.